server/code/setup/cpdb

```diff
@@ -262,8 +262,12 @@ if __name__ == "__main__":
         print("ERROR: --drop can not be used with --schema-only")
         sys.exit(1)
 
+    if not options.ssl and (options.sslfactory or options.sslcertpath):
+        print("ERROR: --sslfactory or --sslcertpath must only be specified with --ssl.")
+        sys.exit(1)
+
     if options.ssl and options.sslfactory and options.sslcertpath:
-        print("ERROR: ssl, sslfactory, and sslcertpath are not supported together.")
+        print("ERROR: --ssl, --sslfactory, and --sslcertpath are not supported together.")
```

Comment:
This message is kind of confusing with the previous one. Since the previous check already verifies either sslfactor or sslcertpath are set with ssl, you can drop that part of the check from here and this message.

Reply:
Thinking on it a bit, it might be cleaner to just combine these two into one check:

```
if options.ssl:
  if options.sslfactory and options.sslcertpath:
    # error
elif options.sslfactory or options.sslcertpath:
  # error

Reply:
Yes. We can combine it into one block as suggested. 
Thanks Chris.

Reply:
Updated the changes.